modules/store/spec/types/feature_creator.spec.ts

```diff
@@ -178,9 +178,9 @@ describe('createFeature()', () => {
           reducer: createReducer(
             initialState,
             on(enter, (state) => ({ ...state, loadState: 'loading' })),
-            on(loadBooksSuccess, (state, { books }) => ({
+            on(loadBooksSuccess, (state, { entities }) => ({
               ...state,
-              books,
+              entities,
```

Comment:
```suggestion
              books: entities,
```

or switch `entities` with `books` (line 152)